service/__init__.py

```diff
@@ -1,8 +1,15 @@
-import service.tasks.accounts
-import service.tasks.monitoring
-import service.tasks.driver
-import service.tasks.volume
-import service.tasks.machine
-import service.tasks.snapshot
-import service.tasks.admin
-import chromogenic.tasks
+import django
+try:
+    #FIXME: Find a way to make tasks 'visible' without abusing the __init__ files
```

Comment:
Is this FIXME still relevant? Is it the last `chromogenic.tasks` thats triggering this?